### PR TITLE
instance list v2

### PR DESCRIPTION
**Pull request by ramitos**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our [guidelines](https://github.com/yldio/joyent-portal/blob/issue-template/.github/COMMIT_GUIDELINES.md)
- [x] Tests for the changes have been added (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

**Does this PR close an issue?** (If not please create one)
#871

**What is the new behavior (if this is a feature change)?**
new table, new bulk actions, new list header, new quick actions

### CODE REVIEW

File: packages/ui-toolkit/src/form/base/input.js [hunk 1]
```diff
@@ -41,13 +41,13 @@ const style = css`
   color: ${color};
 
   &::-webkit-input-placeholder {
-    color: rgba(73, 73, 73, 0.5);
+    color: ${props => props.theme.grey};
```
Comment (SaraVieira) [inline packages/ui-toolkit/src/form/base/input.js]: you can't change this because the color has 0.5 opacity

File: packages/ui-toolkit/src/icon-button/index.js [hunk 2]
```diff
@@ -1,128 +0,0 @@
-import React from 'react';
```
Comment (SaraVieira) [inline packages/ui-toolkit/src/icon-button/index.js]: good

File: packages/ui-toolkit/src/layout/view-container.js [hunk 3]
```diff
@@ -5,8 +5,10 @@ import is, { isNot } from 'styled-is';
 import { styled as breakpoints } from '../breakpoints';
 
 export default Grid.extend`
+  box-sizing: border-box;
+  width: 100%;
+
   ${is('fluid')`
-    width: 100%;
```
Comment (SaraVieira) [inline packages/ui-toolkit/src/layout/view-container.js]: why did you remove this ?

File: packages/ui-toolkit/src/table/index.js [hunk 4]
```diff
@@ -64,14 +69,7 @@ const Column = css`
 
   ${is('actionable')`
     cursor: pointer;
-
```
Comment (SaraVieira) [inline packages/ui-toolkit/src/table/index.js]: why was this removed ?

File: packages/ui-toolkit/src/table/index.js [hunk 5]
```diff
@@ -190,8 +189,6 @@ const BaseTbody = styled.tbody`
 const BaseTh = styled.th`
   ${Column};
 
-  text-align: left;
-  padding: ${remcalc(12)} ${remcalc(24)};
```
Comment (SaraVieira) [inline packages/ui-toolkit/src/table/index.js]: and this ?

File: packages/ui-toolkit/src/table/index.js [hunk 6]
```diff
@@ -253,10 +243,6 @@ const BaseTr = styled.tr`
 
   ${is('selected')`
     box-shadow: none;
-    td {
```
Comment (ramitos) [inline packages/ui-toolkit/src/table/index.js]: confirm
